File: 백준/Bronze/10870. 피보나치 수 5/피보나치 수 5.py
def fibonach(n):
    result = 0
    if n == 0:
        result = 0
    if n == 1:
        result = 1 
    else:    
        if n > 1:
            result = fibonach(n-1) + fibonach(n-2)
    return result
n = int(input())
print(fibonach(n))

File: 백준/Bronze/10870. 피보나치 수 5/피보나치 수 5.c
#include <stdio.h>
int fibonach(int n){
    if (n == 0) return 0;
    else if (n == 1) return 1;
    
    return fibonach(n - 1) + fibonach(n - 2);
    
}

int main(void){
    int n;
    int answer;
    scanf("%d", &n);
    printf("%d\n", fibonach(n));
}